src/components/header/Header.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components'
+
+function RenderHeader() {
+  return (
+    <Header>
+      <h1>Allaround Family</h1>
+    </Header>
+  )
+}
+
+const Header = styled.div`
+  width: 100%;
+  padding: 20px 0px;
+  text-align: center;
+  z-index: 2;
+  position: fixed;
+  top: 0;
+`
```

Comment:
You'll probably need a background (other than transparent) when scrolling. 

Tip: Experiment with `backdrop-filter` e. g. `backdrop-filter: blur(10px)`